MAUI/ListView/getting-started.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Getting Started with .NET Maui ListView (SfListView)
 
-This section provides a quick overview of how to get started with the .NET Maui ListView (SfListView) for Maui. Walk-through the entire process of creating the real world SfListView.
+This section provides a quick overview of how to get started with the [.NET Maui ListView](https://www.syncfusion.com/maui-controls/maui-listview) (SfListView) for Maui. Walk-through the entire process of creating the real world SfListView.
```

Comment:
Don't interlink here, instead you can combine both FT and example in N> section.

Reply:
Resolved